README.md

```diff
@@ -48,6 +48,19 @@ end
 
 Use the batch execution strategy with your schema
```

Comment:
This sentence is describing the old way changing the execution strategy, which isn't applicable to the lazy_resolve approach.